src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -754,6 +790,7 @@ function refreshFromPlayer(player, type) {
 }
 
 function bindToPlayer(player) {
+    lyricPageActive = appRouter.currentRouteInfo.path === '/Lyrics';
```

Comment:
Not the best way to check because the paths are case insensitive.

Reply:
Quick fix. Lyrics toggling doesn't work.
```suggestion
    lyricPageActive = appRouter.currentRouteInfo.path.toLowerCase() === '/lyrics';
```